xline-test-utils/src/bin/test_cluster.rs

```diff
@@ -0,0 +1,18 @@
+use tokio::signal;
+use xline_test_utils::Cluster;
+
+#[tokio::main]
+async fn main() {
+    let mut cluster = Cluster::new(3).await;
+    cluster.start().await;
+
+    println!("cluster running");
+
+    for (id, addr) in cluster.addrs() {
+        println!("server id: {} addr: {}", id, addr);
+    }
+
+    if let Err(e) = signal::ctrl_c().await {
+        eprintln!("Unable to listen for shutdown signal: {e}");
+    }
+}
```

Comment:
Why would a utils crate have a binary executable?

Reply:
For convenient testing, I use it to quickly setup a test cluster for command line client tests.

Reply:
Then add the explanation in the comments so that other won't ask the same question again, like me ;)

Reply:
done